configure.ac

```diff
@@ -1129,6 +1129,12 @@ if test "x${JEMALLOC_PREFIX}" = "x" ; then
   AC_CHECK_FUNC([__libc_free],
 		[AC_DEFINE([JEMALLOC_OVERRIDE___LIBC_FREE], [ ], [ ])
 		 wrap_syms="${wrap_syms} __libc_free"])
+  AC_CHECK_FUNC([__libc_free_sized],
```

Comment:
I suppose this isn't harmful, but I don't think we yet have any indication that glibc will implement these functions under these names.

Reply:
I just spent a few minutes perusing the glibc source, and as far as I can tell from that quick survey their allocator doesn't even implement something akin to `free_sized`. So I'll just remove these.

Reply:
Actually I say just keep it for now. Like David mentioned it's no harm. But worth adding some comments saying that in case the glibc naming is different this needs to be updated.